src/lib/layouts/workflow-history-layout.svelte

```diff
@@ -39,7 +41,18 @@
     run: workflow.runId,
   };
 
+  $: {
+    if ($workflowRun.workflow.isRunning) {
+      clearInterval(refreshInterval);
+      // Auto-refresh of 15 seconds
+      refreshInterval = setInterval(() => ($refresh = Date.now()), 15000);
```

Comment:
Hi, I recommend making this optional. It can be very annoying because the UI has components with states.

Reply:
Yup will be an opt-in feature, have the designs for it now and it will be included in this PR.

Reply:
Awesome